Palace.xcodeproj/project.pbxproj

```diff
@@ -3920,6 +3920,7 @@
 				CLANG_WARN_NON_LITERAL_NULL_CONVERSION = YES_ERROR;
 				CODE_SIGN_ENTITLEMENTS = Palace/SimplyE.entitlements;
 				CODE_SIGN_IDENTITY = "iPhone Developer";
+				CURRENT_PROJECT_VERSION = 47;
 				CURRENT_PROJECT_VERSION = 46;
```

Comment:
It seems Xcode created two build numbers, it does that sometimes, could you try to change build number to some other and then change back? This usually helps.